ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/EventsTableFlatEventsRangeQueries.scala

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.store.dao.events
+
+import anorm.{Row, SimpleSql, SqlStringInterpolation}
+import com.daml.ledger.participant.state.v1.Offset
+import com.digitalasset.daml.lf.data.Ref.Identifier
+import com.digitalasset.platform.store.Conversions._
+
+private[events] sealed trait EventsTableFlatEventsRangeQueries[Offset] {
```

Comment:
Do you plan on having multiple implementations of this trait, or can this just be folded into the accompanying `object`?

Reply:
The second implementation will contain the queries to serve the active contracts.

Reply:
👍